update_prevalence.py

```diff
@@ -281,16 +281,53 @@ def recent_daily_cumulative_cases(self) -> List[int]:
     # cases. Nominally is values[-1] - values[0], but sometimes regions post
     # corrections which result in the number of cases decreasing.
     def cases_in_cum_cases(self, values) -> int:
-        min_index = values.index(min(values))
-        return max(values[min_index:]) - min(values)
+        # list of indices right before negative corrections. If values = [3,2,3,0,5],
+        # then negative_corrections == [0, 2]
+        negative_corrections = [i for i, val in enumerate(values[:-1]) if val > values[i+1]]
+        corrections_within_bounds = min(values) == values[0] and max(values) == values[-1]
+        if len(negative_corrections) == 0 or corrections_within_bounds:
+            return values[-1] - values[0]
+
+        # Always use values[-1] rather than max(values) because max(values) is
+        # either values[-1] or there's been a negative correction that should take precedence.
+        if min(values[:-1]) <= values[-1]:
+            possibly_suspect_correction = False
+            for local_max_idx in negative_corrections:
```

Comment:
nit: prefer `negative_correction_index` instead of `local max_idx`, since as you pointed out, we sometimes see negative cases multiple days in a row.